src/Yoti.Auth/ActivityDetail.cs

```diff
@@ -12,20 +12,42 @@ public enum ActivityOutcome { Success, ProfileNotFound, Failure, SharingFailure
     /// </summary>
     public class ActivityDetails
     {
+        public ActivityDetails(ActivityOutcome activityOutcome)
```

Comment:
Just curious - why do we need two constructors?

Reply:
If the share is successful we use one with 4 args, if it's not successful, we use the other one with just the outcome, this was just a way of making the object immutable after being constructed.